testsuite/test_blobs.py

```diff
@@ -1,11 +1,11 @@
-from pathlib import Path
+# flake8: noqa
```

Comment:
I think we should have another `.flake8` under `testsuite` that excludes more rules, like:

- max line lengths
- star imports

And remove this file ignore.